### PR TITLE
POL-151 - Update Policies to use the Credential Service

### PR DESCRIPTION
**Pull request by nshivappa**
Updated Policies to use the Credential Service

### Description

Updated Policies to use the Credential Service

### Issues Resolved

* Replaced auth declaration with the credential declaration
 * Removed long_description field and use new info hash for Version, providers, and service.
 * Updated tenant to single
 * Update readme to replace credential instructions with instructions to use credential service.
 * update CWF definitions as needed.
 * Add info.
 * Remove tenant id and subscription id and iterate over all subscriptions for Azure policies. 

### Contribution Check List

- [ ] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

###  Running template link
AWS Open Buckets:
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5dfb6756372f7a0001f6db0d

AWS Unused RDS Instances:
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5dfb5f24372f7a0001f6daeb

AWS Rightsize RDS Instances:
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5dfb5e551e9a8800018b69f2

Azure Rightsize SQL Databases:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5dfabc34bbbcc0000108ecc5

Azure Subscription Access:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5dfac35cf47a460001252b26

Azure Long Stopped Instances:
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5dfa009ff47a4600012529b3

### CODE REVIEW

**Comment by nshivappa** [comment]
@cdwilhelm,
Addressed some Danger ERRORs other than 

1. The URL is not valid: https://docs.rightscale.com/credentials/ in security/storage/aws/public_buckets/README.md Status: 404

- ERROR is there for All 6 policies since the "https://docs.rightscale.com/credentials/" is not available we are getting this ERROR

1. security/storage/aws/public_buckets/CHANGELOG.md:2: MD009 Trailing spaces

- Getting this ERROR even though there is no Trailing space.

**Comment by cdwilhelm** [comment]
@nshivappa   you can ignore the errors for https://docs.rightscale.com/credentials/.  The docs are not published yet.  We won't release these PR until the cred service is released.
It looks like there is a bug with the MarkDownLink and the way windows handles new lines.  We will update the gem to resolve the trailing space issue. 

Please fix these errors 
security/storage/aws/public_buckets/CHANGELOG.md:43: MD022 Headers should be surrounded by blank lines

File: security/storage/aws/public_buckets/README.md [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
 The policy includes optional support to send a notification to a slack channel when an anomaly is detected.
 The policy accepts two optional input parameters to support this capability:
+
 - `Slack Channel Name`: This is the slack channel name, e.g. #policy_alerts
 - `RightScale Credential for Slack Channel Webhook`: This is the name of a RightScale credential in the account that contains the Slack webhook URL.
 
-### Required Permissions
+## Required Permissions
+
+### Required RightScale Roles
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/README.md]: No rightscale roles are needed
remove text "The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. "

File: security/storage/aws/public_buckets/aws_public_buckets.pt [hunk 2]
```diff
@@ -2,9 +2,16 @@ name "AWS Open Buckets"
...
+long_description ""
 severity "high"
 category "Security"
+tenancy "single"
+info({
+  version: "2.0",
+  provider: "AWS",
+  service: "S3",
+  policy_set: "aws-open-buckets"
+})
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/aws_public_buckets.pt]: Same as above

File: security/storage/aws/public_buckets/aws_public_buckets.pt [hunk 3]
```diff
@@ -28,11 +35,12 @@ parameter "param_slack_webhook_cred" do
...
-auth "auth_aws", type: "aws" do
-  version 4
-  service 's3'
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "AWS IAM"
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/aws_public_buckets.pt]: Same as above

**Comment by cdwilhelm** [comment]
@nshivappa I updated Danger to fix the trailing space issue.  There are still some bad URLs.  The docs.rightscale.com/credentials can be ignored for now.

File: compliance/azure/azure_long_stopped_instances/README.md [hunk 4]
```diff
@@ -12,29 +12,22 @@ The policy leverages the Azure API to check all instances that have been stopped
...
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
+## Required Permissions
 
-## Required RightScale Roles
+### Required RightScale Roles
 
 - Cloud Management - credential_viewer
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/README.md]: Removed rightscale roles

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 5]
```diff
@@ -5,6 +5,13 @@ short_description "Checks for Azure instances that have been stopped for more th
 long_description "Version: 1.3"
 category "Compliance"
 severity "low"
+tenancy "single"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: OKay.
@cdwilhelm : Tenancy "single" has to be removed in all policies or only for this policy?

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 6]
```diff
@@ -5,6 +5,13 @@ short_description "Checks for Azure instances that have been stopped for more th
...
+info({
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: removed.

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 7]
```diff
@@ -5,6 +5,13 @@ short_description "Checks for Azure instances that have been stopped for more th
 long_description "Version: 1.3"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: Sorry, I missed it. Removed now.

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 8]
```diff
@@ -49,15 +42,12 @@ end
...
-    client_secret cred("AZURE_APPLICATION_KEY")
-    additional_params do {
-      "resource" => "https://management.azure.com/"
-    } end
-  end
+#authenticate with Azure
+credentials "azure_auth" do
+  schemes "oauth2"
+  label "Azure"
+  description "Azure Resource Manager"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: changed.

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 9]
```diff
@@ -165,10 +182,10 @@ policy "azure_resource_policy" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure VMs in Stopped State for Over {{parameters.param_numberofdays}} Day(s)"
     detail_template <<-EOS
 # Azure Virtual Machines Out of Compliance
-| Name | Resource Group | Status | Time Stopped | Days Elapsed |
-| ---- | -------------- | ------ | ------------ | ------------ |
-{{ range data -}}
-| {{.name}} | {{.resourcegroup}} | {{.status}} | {{.time_stopped}} | {{.days_elapsed}} |
+| Subscription Name | Name | Resource Group | Status | Time Stopped | Days Elapsed |
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: It is in First column only.

File: compliance/azure/subscription_access/README.md [hunk 10]
```diff
@@ -9,33 +9,34 @@ This policy checks all users who have Owner or Contributor access to a given Azu
...
+## Required Permissions
 
-## Required RightScale Roles
+### Required RightScale Roles
 
+- policy_designer
+- policy_manager
+- policy_publisher
 - credential_viewer
+- observer
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/subscription_access/README.md]: removed rightscale roles

File: compliance/azure/subscription_access/azure_subscription_access.pt [hunk 11]
```diff
@@ -2,9 +2,16 @@ name "Azure Subscription Access"
...
+long_description ""
 category "Compliance"
 severity "low"
+tenancy "single"
+info({
+  version: "2.0",
+  provider: "Azure",
+  service: "Subscription",
+  policy_set: "azure-subscription-access"
+})
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/subscription_access/azure_subscription_access.pt]: removed.

File: compliance/azure/subscription_access/azure_subscription_access.pt [hunk 12]
```diff
@@ -50,26 +43,19 @@ end
...
-    client_secret cred("AZURE_APPLICATION_KEY")
-    additional_params do {
-      "resource" => "https://management.azure.com/"
-    } end
-  end
+#authenticate with Azure
+credentials "azure_auth" do
+  schemes "oauth2"
+  label "Azure"
+  description "Azure Resource Manager"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/subscription_access/azure_subscription_access.pt]: Updated.

File: compliance/azure/subscription_access/azure_subscription_access.pt [hunk 13]
```diff
@@ -225,10 +234,10 @@ policy "azure_resource_policy" do
     summary_template "{{ len data }} Azure Users who are {{parameters.param_azure_roles}}"
     detail_template <<-EOS
 # Azure users out of compliance
-| Resource Name | Azure Username | Azure User Principal Name | Role |
-| ------------- | -------------- | ------------------------- | ---- |
+| Subscription Name | Resource Name | Azure Username | Azure User Principal Name | Role |
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/subscription_access/azure_subscription_access.pt]: It is in first column only.

File: cost/aws/rds_instance_cloudwatch_utilization/README.md [hunk 14]
```diff
@@ -24,10 +20,39 @@ This policy has the following input parameters required when launching the polic
...
+
+## Required Permissions
+
+### Required RightScale Roles
+
+- policy_designer
+- policy_manager
+- policy_publisher
+- credential_viewer
+- observer
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/rds_instance_cloudwatch_utilization/README.md]: Removed.

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt [hunk 15]
```diff
@@ -5,6 +5,13 @@ short_description "Checks for Azure instances that have been stopped for more th
 long_description "Version: 1.3"
 category "Compliance"
 severity "low"
+tenancy "single"
+info({
+  version: "2.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: "azure-long-stopped-instances"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: updated.

File: cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt [hunk 16]
```diff
@@ -2,9 +2,16 @@ name "AWS Rightsize RDS Instances"
...
-long_description "Version: 1.1"
+long_description ""
 severity "low"
 category "Cost"
+tenancy "single"
+info({
+  version: "2.0",
+  provider: "AWS",
+  service: "RDS",
+  policy_set: "aws-rightsize-rds-instance"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt]: updated.

File: cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt [hunk 17]
```diff
@@ -53,18 +60,12 @@ end
...
-auth "monitoring_auth_aws", type: "aws" do
-  version 4
-  service "monitoring"
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "AWS IAM"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt]: Updated.

File: cost/aws/unused_rds/README.md [hunk 18]
```diff
@@ -1,24 +1,44 @@
...
+
+## Required Permissions
+
+### Required RightScale Roles
+
+- policy_designer
+- policy_manager
+- policy_publisher
+- credential_viewer
+- observer
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/unused_rds/README.md]: removed rightscale roles

File: cost/aws/unused_rds/unused_rds.pt [hunk 19]
```diff
@@ -2,9 +2,16 @@ name "AWS Unused RDS Instance"
...
-long_description "Version: 1.0"
+long_description ""
 severity "low"
 category "Cost"
+tenancy "single"
+info({
+  version: "2.0",
+  provider: "AWS",
+  service: "RDS",
+  policy_set: "aws-unused-rds-instance"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/unused_rds/unused_rds.pt]: Updated.

[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/unused_rds/unused_rds.pt]: Removed.

File: cost/aws/unused_rds/unused_rds.pt [hunk 20]
```diff
@@ -36,18 +43,12 @@ end
...
-auth "monitoring_auth_aws", type: "aws" do
-  version 4
-  service "monitoring"
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "AWS IAM"
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/unused_rds/unused_rds.pt]: Updated.

File: cost/aws/unused_rds/unused_rds.pt [hunk 21]
```diff
@@ -278,7 +279,7 @@ ___
 EOS
     escalate $email
     escalate $decommision_rds
-    check gt(to_n(val(item,"connections")),0)
+    check gt(to_n(val(item,"connections")),1000)
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/aws/unused_rds/unused_rds.pt]: Sorry, it was my mistake.
I had changed it for testing.
I removed it now.

File: cost/azure/rightsize_sql_instances/README.md [hunk 22]
```diff
@@ -17,32 +17,25 @@ This policy checks all the Azure SQL databases for a Azure Subscription. It does
...
 
-## Actions
+The credential must contain the value *Azure RM* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
 
-- Sends an email notification
-- Rightsize SQL Databases after approval
+## Required Permissions
 
 ### Required RightScale Roles
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/azure/rightsize_sql_instances/README.md]: Removed.

File: cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt [hunk 23]
```diff
@@ -302,10 +319,10 @@ policy 'policy_azure_db_utilization' do
   - Downsize Average CPU% threshold : {{ parameters.param_avg_cpu_downsize }}
 
 
-| Region | Name | Resource Group | Kind | Type | Current Capacity | Recommended Capacity | SKU - Name | SKU - Tier | CPU Average % | Recommendation |
-| ------ | ---- | -------------- | ---- | ---- | ---------------- | -------------------- | ---------- | ---------- | ------------- | -------------- |
+| Subscription Name | Region | Name | Resource Group | Kind | Type | Current Capacity | Recommended Capacity | SKU - Name | SKU - Tier | CPU Average % | Recommendation |
```
[the remark this answers is not in the document]
Comment (nshivappa) [reply cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt]: It is in the first column.

File: security/storage/aws/public_buckets/aws_public_buckets.pt (hunk 2 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/aws_public_buckets.pt]: Updated

File: security/storage/aws/public_buckets/README.md (hunk 1 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/README.md]: Updated.

File: security/storage/aws/public_buckets/aws_public_buckets.pt (hunk 3 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/aws_public_buckets.pt]: Updated.

File: compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt (hunk 5 above)
[the remark this answers is not in the document]
Comment (cdwilhelm) [reply compliance/azure/azure_long_stopped_instances/long_stopped_instances_azure.pt]: all policies that use the credential service.  when using the credential service. the policy is only single tenant, it's hard coded in the backend and doesn't need to be added here.

File: security/storage/aws/public_buckets/aws_public_buckets.pt [hunk 24]
```diff
@@ -28,11 +34,12 @@ parameter "param_slack_webhook_cred" do
...
-auth "auth_aws", type: "aws" do
-  version 4
-  service 's3'
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "Select the AWS IAM Credential from the list."
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/aws_public_buckets.pt]: Remove IAM

File: security/storage/aws/public_buckets/README.md [hunk 25]
```diff
@@ -1,38 +1,43 @@
...
+## Policy Actions
 
 The following policy actions are taken on any resources found to be out of compliance.
 
 - Send an email report
 
-#### Slack Channel Notification Support
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/README.md]: Remove IAM User

File: cost/aws/rds_instance_cloudwatch_utilization/README.md [hunk 26]
```diff
@@ -24,10 +20,29 @@ This policy has the following input parameters required when launching the polic
...
+## Policy Actions
+
+The following policy actions are taken on any resources found to be out of compliance.
+
+- Send an email report
+- Resizes RDS Instances after approval.
+
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
```
Comment (cdwilhelm) [inline cost/aws/rds_instance_cloudwatch_utilization/README.md]: Remove IAM User

File: cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt [hunk 27]
```diff
@@ -53,18 +59,12 @@ end
...
-auth "monitoring_auth_aws", type: "aws" do
-  version 4
-  service "monitoring"
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "Select the AWS IAM Credential from the list."
```
Comment (cdwilhelm) [inline cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt]: Remove IAM

File: cost/aws/unused_rds/README.md [hunk 28]
```diff
@@ -1,29 +1,39 @@
...
 
-### AWS Required Permissions
+The following policy actions are taken on any resources found to be out of compliance.
+
+- Send an email report
+- decommissions AWS RDS instances after approval.
+
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
```
Comment (cdwilhelm) [inline cost/aws/unused_rds/README.md]: Remove IAM User

File: cost/aws/unused_rds/README.md [hunk 29]
```diff
@@ -1,29 +1,39 @@
...
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
+
+The credential must contain the value *AWS* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
+
+## AWS Required Permissions
 
 This policy requires permissions to list Metrics and Get Metric Statistics from the AWS Cloudwatch API.
-The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
+The IAM user credentials requires the following permissions:
```
Comment (cdwilhelm) [inline cost/aws/unused_rds/README.md]: Remove IAM

File: cost/aws/unused_rds/unused_rds.pt [hunk 30]
```diff
@@ -36,18 +42,12 @@ end
...
-auth "monitoring_auth_aws", type: "aws" do
-  version 4
-  service "monitoring"
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "Select the AWS IAM Credential from the list."
```
Comment (cdwilhelm) [inline cost/aws/unused_rds/unused_rds.pt]: Remove IAM

File: security/storage/aws/public_buckets/README.md [hunk 31]
```diff
@@ -1,38 +1,43 @@
...
-
-- Cloud Management - credential_viewer or admin
-- Cloud Management - Observer
-
-### AWS Required Permissions
+## AWS Required Permissions
 
 This policy requires permissions to describe AWS S3 ListAllMyBuckets, GetBucketLocation and GetBucketAcl.
-The Cloud Management Platform automatically creates two Credentials when connecting AWS to Cloud Management; AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. The IAM user credentials contained in those credentials will require the following permissions:
+The IAM user credentials requires the following permissions:
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/README.md]: Remove IAM

File: security/storage/aws/public_buckets/aws_public_buckets.pt (hunk 24 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/aws_public_buckets.pt]: Removed.

File: security/storage/aws/public_buckets/README.md (hunk 25 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/README.md]: Updated

File: cost/aws/rds_instance_cloudwatch_utilization/README.md (hunk 26 above)
Comment (nshivappa) [reply cost/aws/rds_instance_cloudwatch_utilization/README.md]: Updated.

File: cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt (hunk 27 above)
Comment (nshivappa) [reply cost/aws/rds_instance_cloudwatch_utilization/rds_instance_cloudwatch_utilization.pt]: Updated.

File: cost/aws/unused_rds/README.md (hunk 28 above)
Comment (nshivappa) [reply cost/aws/unused_rds/README.md]: Updated

File: cost/aws/unused_rds/README.md (hunk 29 above)
Comment (nshivappa) [reply cost/aws/unused_rds/README.md]: Updated.

File: cost/aws/unused_rds/unused_rds.pt (hunk 30 above)
Comment (nshivappa) [reply cost/aws/unused_rds/unused_rds.pt]: Updated

File: compliance/azure/subscription_access/azure_subscription_access.pt [hunk 32]
```diff
@@ -1,10 +1,16 @@
...
-long_description "Version: 1.3"
+short_description "Lists anyone who has been granted Owner or Contributor access to an Azure subscription. \n See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/azure/subscription_access) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
 category "Compliance"
 severity "low"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Identity",
+  policy_set: "Azure Subscription Access"
```
Comment (cdwilhelm) [inline compliance/azure/subscription_access/azure_subscription_access.pt]: this policy template doesn't have a policy set

File: security/storage/aws/public_buckets/aws_public_buckets.pt [hunk 33]
```diff
@@ -1,10 +1,16 @@
...
-long_description "Version: 1.9"
+short_description "Check for buckets that are open to everyone. See the [README](https://github.com/flexera/policy_templates/tree/master/security/storage/aws/public_buckets) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
 severity "high"
 category "Security"
+info(
+  version: "2.0",
+  provider: "AWS",
+  service: "S3",
+  policy_set: "aws-open-buckets"
```
Comment (cdwilhelm) [inline security/storage/aws/public_buckets/aws_public_buckets.pt]: policy set should be "Open Buckets"

File: compliance/azure/azure_long_stopped_instances/README.md [hunk 34]
```diff
@@ -12,27 +12,14 @@ The policy leverages the Azure API to check all instances that have been stopped
...
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
-
-## Required RightScale Roles
-
-- Cloud Management - credential_viewer
+The credential must contain the value *Azure RM* in the Provider field.
```
Comment (nshivappa) [inline compliance/azure/azure_long_stopped_instances/README.md]: @cdwilhelm whether it should be *Azure RM* or *azure_rm*.

File: compliance/azure/subscription_access/azure_subscription_access.pt (hunk 32 above)
Comment (nshivappa) [reply compliance/azure/subscription_access/azure_subscription_access.pt]: Update.

File: security/storage/aws/public_buckets/aws_public_buckets.pt (hunk 33 above)
Comment (nshivappa) [reply security/storage/aws/public_buckets/aws_public_buckets.pt]: Updated, Thanks.

File: compliance/azure/azure_long_stopped_instances/README.md (hunk 34 above)
Comment (cdwilhelm) [reply compliance/azure/azure_long_stopped_instances/README.md]: Thanks for mentioning this.  it was intended to have the UI format Azure RM to azure_rm, but it's not implemented.  so, yes please change to azure_rm.

File: cost/aws/rds_instance_cloudwatch_utilization/README.md [hunk 35]
```diff
@@ -24,10 +20,29 @@ This policy has the following input parameters required when launching the polic
...
+
+This policy requires the AWS Credential. When applying the policy select the appropriate credentials
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
+
+The credential must contain the value *AWS* in the Provider field.
+Refer to our documentation for more details on the [Credential Service](https://docs.rightscale.com/credentials/)
+
+## AWS Required Permissions
+
+- AWS - Read access to CloudWatch & RDS
```
Comment (ryanoleary) [inline cost/aws/rds_instance_cloudwatch_utilization/README.md]: Can we list the actual policy JSON for this?

**Pull request merged**